graphql-ballerina/modules/parser/document_node.bal

```diff
@@ -31,12 +31,30 @@ public class DocumentNode {
     public isolated function addFragment(FragmentNode fragment) returns SyntaxError? {
         if (self.fragments.hasKey(fragment.getName())) {
             FragmentNode originalFragment = <FragmentNode>self.fragments[fragment.getName()];
-            string message = string`There can be only one fragment named "${fragment.getName()}".`;
-            Location l1 = originalFragment.getLocation();
-            Location l2 = fragment.getLocation();
-            self.errors.push({message: message, locations: [l1, l2]});
+            if(fragment.isIncludeInlineFragment()) {
```

Comment:
This name is not clear. A better name would be `isInlineFragment`. (We want to check whether this is an inline fragment or not right?).